packages/lesswrong/components/posts/PostsItem2.jsx

```diff
@@ -407,7 +410,12 @@ const PostsItem2 = ({
             </PostsItem2MetaInfo>
 
             <span className={classNames(classes.title, {[classes.hasSmallSubtitle]: (!!resumeReading || showNominationCount)})}>
-              <PostsTitle postLink={postLink} post={post} expandOnHover={!renderComments} read={isRead && !renderComments} sticky={isSticky(post, terms)} showQuestionTag={showQuestionTag}/>
+                <AnalyticsTracker
+                    eventType={"postItem"}
+                    eventProps={{postId: post._id, isSticky:isSticky(post, terms)}}
+                    captureOnMount={(eventData => captureOnMountContexts.includes(eventData.listContext))}>
```

Comment:
I don't think capturing mount events for individual post items makes sense. I'd much rather this be done at the PostsList level; that will generate an order of magnitude less raw events for the same info.

Reply:
PostList events fire where PostLists are used, but in several places they're not, e.g. BookMarks, FromTheArchives, ContinueReading. Those are constructed with map functions over a list of posts to PostItem2 components.